### ISSUE TITLE
List ideologies in the results and add match percentage

### ISSUE THREAD

**Issue by Ardutronico** [8values/8values.github.io#78]
Hello, the problem is that the various ideologies and their percentages can't be known without opening the file ideologies.js, and this makes comparison very difficult and you don't know how right the test was. At least add the percentage of the match to the closest ideology. I'd also like that there was an arrow that when clicked showed an element with the percentage of the match to the other ideologies ordered from higher to lower.

**Comment by Berkmann18** [comment]
I completely agree, it's hard to know the accuracy of the result evaluation without scrutinating the code.
And sometimes one might be close to two (or more, although unlikely) ideologies.

**Comment by endolith** [comment]
Yeah, ISideWith.com shows a list of your best-matching ideologies and how well you match them.
